### PR TITLE
docs: add more details to README

### PR DESCRIPTION
**Pull request by zeke**
I took another pass at the README, attempting to make it more clear how this project works and all the steps required to set it up.

Rendered: https://github.com/repo-sync/repo-sync/blob/ef4b7f80b0ddd62783a45db56a018992d54f00f7/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,52 +3,45 @@
...
 > Keep a pair of GitHub repos in sync.
 
-:bulb::construction::bulb: Work in progress. Use at your own risk! :bulb::construction::bulb:
+## How it Works
+
+This project uses [GitHub Actions](https://github.com/features/actions) workflows to keep pairs of git repos in sync. It runs on a [schedule](#cron) (every 15 minutes by default). Shortly after changes are made to the default branch of **repo A**, the Actions workflow runs on **repo B** and generates a pull request including the recent changes from **repo A**. If more changes are made to **repo A** before the pull request is merged, those changes will be added to the existing pull request. The same is true in the opposite direction: changes made to **repo B** will eventually get picked up by the workflow in **repo A**. 
 
 ## Features
 
-- Sync from a repo
```
Comment (wei) [inline README.md]: Do we want to drop the uni-direction use-case and only highlight the bi-direction use-case?

File: README.md [hunk 2]
```diff
@@ -3,52 +3,45 @@
...
-
-The easiest way to set up Repo Sync is using the [interactive installation page](https://github-repo-sync.herokuapp.com). This page will ask you to authenticate with your GitHub account. Fill out the form, specifying source repo, target repo, and an intermediate branch name to use for pull requests. Once you've submitted this form, the page will generate [Actions workflow files](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file) for you, and can optionally automatically commit them directly to your repository.
-
 ## Manual Installation
 
-If you'd prefer to set up your Actions workflows manually, that's also an option:
+### Step 1. Set up Secrets
 
-### 1. Set up secrets
+[GitHub Secrets] are environment variables stored on your GitHub repository that are made available in the GitHub Actions environment. There are two (2) secrets are required on each repo. Go to Settings > Secrets on your repo page and add the following secrets:
```
Comment (wei) [inline README.md]: Starting with v2, my understanding is secrets are no longer environment variables by default and has to be manually added to environment variables or inputs using `${{ secrets.SOME_SECRET }}`

So maybe just remove the word `environment` here.

File: README.md [hunk 3]
```diff
@@ -3,52 +3,45 @@
...
-If source repo is a public GitHub repo:
+The shorthand name or URL of the repo to sync.
 
-| Name | Example Value | Description |
-| --- | --- | --- |
-| SOURCE_REPO	| `owner/repo` | Repository slug |
-| INTERMEDIATE_BRANCH | `repo-sync` | Branch name for pull requests |
+- If the source repo is a **public** GitHub repo, use a shorthand name like `owner/repo`.
+- If the source repo is a **private** GitHub repo, specify an HTTPS clone URL in the format `https://<access_token>@github.com/owner/repo.git` that includes an access token with `repo` scope. [Generate a token]((https://github.com/settings/tokens/new?description=repo-sync&scopes=repo)).
+- If the source repo is not hosted on GitHub, specify an HTTPS URL that includes push access credentials.
```
Comment (wei) [inline README.md]: We require pull access from source repo not push.

Comment (zeke) [reply README.md]: 🚫 📌

File: README.md (hunk 1 above)
Comment (zeke) [reply README.md]: Good catch. Added "One-way or two-way sync"

**Pull request merged**